src/main/webapp/scripts/tests/view-test.js

```diff
@@ -7,3 +7,7 @@ QUnit.test('createMenu', function(assert) {
   createMenu();
   assert.dom('.nav-item').exists({ count: 3});
 });
+
+QUnit.test('iFrame created', function(assert) {
+  assert.dom('iframe').exists({ count: 1 });
+});
```

Comment:
Should I be testing that the iframe is added? honestly not sure what to test here, because the iframe api adds it, and I could mock the API, but then there is no code to test...hmm

Reply:
good enough for me lol.